devel/README.md

```diff
@@ -80,16 +80,17 @@ for how to do this for your platform.
 ### Setting up environment variables
 
 Set up your GOPATH, add a path entry for Go binaries to your PATH, set the ISTIO
-path, and set your GITHUB_USER used later in this document.  These exports are
+path, and set your GITHUB_USER used later in this document. These exports are
 typically added to your ~/.profile:
 
 ```shell
 export GOPATH=~/go
 export PATH=$PATH:$GOPATH/bin
 export ISTIO=$GOPATH/src/istio.io # eg. ~/go/src/istio.io
 
-# Replace "$GITHUB_USER" below with your github username
-export GITHUB_USER=$USER # replace with actual if different
+# If your github username is not the same as your local user name (saved in the
+# shell variable $USER), then replace "$USER" below with your github username
+export GITHUB_USER=$USER
```

Comment:
I think the original document is clear enough as it already claiming update the "$GITHUB_USER" below with your github username

Reply:
Not sure I totally agree.  line 91 in the original text indicates replacing $GITHUB_USER (which = $USER) although it is possible someone might read that as "Replace "GITHUB_USER" below with your github username.

As an example, lets say the machine login is stdake and the github login is sdake, I could see people doing something like this:
export sdake=$USER

Reply:
A different way to fix this would be have the text read
```replace "$USER" below with your github id```

Reply:
+1 for this, @sdake